payment/src/main/resources/db/changelog/data/changelog-0001-provider.sql

```diff
@@ -2,6 +2,6 @@
 
 --changeset hoanglam:paypal
 insert into payment_provider (id, is_enabled, name, configure_url, landing_view_component_name, additional_settings)
-    values ('PaypalPayment', 'true', 'paypal', 'paypal-config', 'paypal landing view', '{"clientId": "AW7GUe26RhVRlWKHeKHjl43ZqON8NFgJbEOljFDkuBiLlFYWj7mskz77QgVMHkl2M9VBMA5jWMFwxRll", "clientSecret": "ENX1js2V-5bdTe86voCAcyrT6bcRtVzRcvNsdo0XDhDYT_5KEBmlYT7oOBWVVwjPsS4i6bktL1R8eL3j", "mode": "sandbox"}');
+    values ('PAYPAL', 'true', 'paypal', 'paypal-config', 'paypal landing view', '{"clientId": "AW7GUe26RhVRlWKHeKHjl43ZqON8NFgJbEOljFDkuBiLlFYWj7mskz77QgVMHkl2M9VBMA5jWMFwxRll", "clientSecret": "ENX1js2V-5bdTe86voCAcyrT6bcRtVzRcvNsdo0XDhDYT_5KEBmlYT7oOBWVVwjPsS4i6bktL1R8eL3j", "mode": "sandbox"}');
```

Comment:
I think we should create changelog-0003-provider.sql and run SQL Update. So that other developer don't need to reset the database

Reply:
thanks, i will make changes